saleor/graphql/schema.graphql

```diff
@@ -3319,6 +3323,8 @@ type ProductVariant implements Node {
   images: [ProductImage]
   translation(languageCode: LanguageCodeEnum!): ProductVariantTranslation
   digitalContent: DigitalContent
+  stocks: Stock
```

Comment:
Idea - why not `stock(country:  CountryCode): Stock` instead of these two queries?